snuba/datasets/metrics_messages.py

```diff
@@ -93,7 +95,7 @@ def aggregation_options_for_set_message(
             GRANULARITY_ONE_DAY,
         ],
         "min_retention_days": retention_days,
-        "materialization_version": 1,
+        "materialization_version": get_config("gen_metric_sets_mv_ver", 1),
```

Comment:
Where is this config defined again? Just wanted to see it again

Reply:
I think it's [here](https://github.com/getsentry/snuba/blob/14a903d6abd207259980268d6947ff83accc2a5e/snuba/state/__init__.py#L190). I'm not actually very certain if we should be doing this. Since get_config will likely incur some performance penalty because it's communicating with something.

Reply:
cc @nikhars on your thoughts here

Reply:
historically we've changed materialization version with a setting (requires deploy to update, stored in memory) rather than run-time config. That would mean that we don't hit Redis to fetch run-time config values, but I think we already have logic in the consumers that caches in memory and only checks for new values once/minute, so I'm not concerned about this approach.

I don't feel strongly about which approach we take, though I do think we should just eventually change this to be hardcoded to 2

Reply:
Cool, thanks